### PR TITLE
Validation for Ov6.0

### PR DESCRIPTION
**Pull request by AsisBagga**
### Description
1. Validation for Ov6.0 
2. Changes in example directory for Test Automations.
3. Updates in resource and data source files of LI for GetByName Feature

### Issues Resolved
N/A

### Check List
- [ ] New functionality includes testing.
  - [ ]  All tests pass for go 1.11 + gofmt checks.
- [ ] New functionality has been documented in the README if applicable.
  - [ ] New functionality has been thoroughly documented in the examples (please include helpful comments).
  - [ ] New endpoints supported are updated in the endpoints-support.md file.
- [ ] Changes are documented in the CHANGELOG.

### CODE REVIEW

File: oneview/data_source_logical_interconnect.go [hunk 1]
```diff
@@ -336,8 +337,14 @@ func dataSourceLogicalInterconnect() *schema.Resource {
 func dataSourceLogicalInterconnectRead(d *schema.ResourceData, meta interface{}) error {
 	config := meta.(*Config)
 	id := d.Get("name").(string)
-	logInt, err := config.ovClient.GetLogicalInterconnectById(id)
 
+	logInt := ov.LogicalInterconnect{}
+	all_li, err := config.ovClient.GetLogicalInterconnects("", "", "")
```
[the remark this answers is not in the document]
Comment (AsisBagga) [reply oneview/data_source_logical_interconnect.go]: done

File: oneview/resource_logical_interconnects.go [hunk 2]
```diff
@@ -347,7 +348,14 @@ func resourceLogicalInterconnect() *schema.Resource {
 func resourceLogicalInterconnectRead(d *schema.ResourceData, meta interface{}) error {
 	config := meta.(*Config)
 	id := d.Id()
-	logInt, err := config.ovClient.GetLogicalInterconnectById(id)
+
+	logInt := ov.LogicalInterconnect{}
+	all_li, err := config.ovClient.GetLogicalInterconnects("", "", "")
```
[the remark this answers is not in the document]
Comment (AsisBagga) [reply oneview/resource_logical_interconnects.go]: done

File: oneview/data_source_logical_interconnect.go (hunk 1 above)
[the remark this answers is not in the document]
Comment (AsisBagga) [reply oneview/data_source_logical_interconnect.go]: done

File: examples/fc_networks/.terraform/plugins/linux_amd64/lock.json [hunk 3]
```diff
@@ -0,0 +1,3 @@
+{
+  "oneview": "9373f8f9b68bd6c5a8eb952b5f75bc68f6eefe9fa0d55d6a31c44b72ac00dd5d"
```
Comment (VenkateshRavula) [inline examples/fc_networks/.terraform/plugins/linux_amd64/lock.json:2]: just asking for my knowledge. Why is this required..?

File: examples/fc_networks/main.tf [hunk 4]
```diff
@@ -0,0 +1,20 @@
+provider "oneview" {
+        ov_username =   "${var.username}"
+        ov_password =   "${var.password}"
+        ov_endpoint =   "${var.endpoint}"
+        ov_sslverify =  "${var.ssl_enabled}"
+        ov_apiversion = 2400
```
Comment (VenkateshRavula) [inline examples/fc_networks/main.tf:6]: change api version

File: examples/fc_networks/update_resource.tf [hunk 5]
```diff
@@ -0,0 +1,15 @@
+provider "oneview" {
+        ov_username =   "${var.username}"
+        ov_password =   "${var.password}"
+        ov_endpoint =   "${var.endpoint}"
+        ov_sslverify =  "${var.ssl_enabled}"
+        ov_apiversion = 2400
```
Comment (VenkateshRavula) [inline examples/fc_networks/update_resource.tf:6]: change api version

File: examples/logical_interconnect_groups/data_source.tf [hunk 6]
```diff
@@ -0,0 +1,21 @@
+provider "oneview" {
+  ov_username =   "${var.username}"
+  ov_password =   "${var.password}"
+  ov_endpoint =   "${var.endpoint}"
+  ov_sslverify =  "${var.ssl_enabled}"
+  ov_apiversion = 2400
```
Comment (VenkateshRavula) [inline examples/logical_interconnect_groups/data_source.tf:6]: change api version

File: examples/logical_interconnect_groups/update_resource.tf [hunk 7]
```diff
@@ -0,0 +1,83 @@
+provider "oneview" {
+  ov_username =   "${var.username}"
+  ov_password =   "${var.password}"
+  ov_endpoint =   "${var.endpoint}"
+  ov_sslverify =  "${var.ssl_enabled}"
+  ov_apiversion = 2400
```
Comment (VenkateshRavula) [inline examples/logical_interconnect_groups/update_resource.tf:6]: change api version

Comment (AsisBagga) [reply examples/logical_interconnect_groups/update_resource.tf]: done

File: examples/logical_interconnect_groups/data_source.tf (hunk 6 above)
Comment (AsisBagga) [reply examples/logical_interconnect_groups/data_source.tf]: done

File: examples/fc_networks/update_resource.tf (hunk 5 above)
Comment (AsisBagga) [reply examples/fc_networks/update_resource.tf]: done

File: examples/fc_networks/main.tf (hunk 4 above)
Comment (AsisBagga) [reply examples/fc_networks/main.tf]: done

File: examples/fc_networks/import.tf [hunk 8]
```diff
@@ -0,0 +1,14 @@
+provider "oneview" {
+        ov_username =   "${var.username}"
+        ov_password =   "${var.password}"
+        ov_endpoint =   "${var.endpoint}"
+        ov_sslverify =  "${var.ssl_enabled}"
+        ov_apiversion = 2400
```
[the remark this answers is not in the document]
Comment (AsisBagga) [reply examples/fc_networks/import.tf]: done

File: CHANGELOG.md [hunk 9]
```diff
@@ -1,14 +1,11 @@
 All notable changes to this project will be documented in this file . 
 This project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html)
 
-
-# [v1.7.0]
+# [v6.0.0-11]
```
[the remark this answers is not in the document]
Comment (AsisBagga) [reply CHANGELOG.md]: done

File: examples/deployment_plans/main.tf [hunk 10]
```diff
@@ -0,0 +1,124 @@
...
+}
+
+# Creates a deployment plan 
+
+resource "oneview_deployment_plan" "dp" {
+	name = "DemoDeploymentPlan"
+	oe_build_plan_uri = "/rest/build-plans/97c12c1a-a048-4b52-bcdb-de7f20b5295f"
+	hp_provided = false
+	type = "OEDeploymentPlanV5"
+	custom_attributes = [ ]
```
Comment (nabhajit-ray) [inline examples/deployment_plans/main.tf:18]: This square brackets  should contain the constraints

**Review by nabhajit-ray — APPROVED** [review]

**Review by VenkateshRavula — APPROVED** [review]
